src/main/java/frc/robot/subsystems/BallPath/Shooter/ShooterImpl.java

```diff
@@ -2,20 +2,85 @@
 
 import java.util.concurrent.TimeUnit;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
 // import ca.team3161.lib.robot.subsystem.RepeatingIndependentSubsystem;
 import ca.team3161.lib.robot.subsystem.RepeatingPooledSubsystem;
+import ca.team3161.lib.utils.Utils;
+// import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class ShooterImpl extends RepeatingPooledSubsystem implements Shooter {
-    public ShooterImpl() {
+    
+    private TalonSRX turretMotor;
+    private TalonFX shooterMotor;
+    private TalonSRX hoodMotor;
+
+    private double kp = 0.00035;
+    private double ki = 0.000075;
+    // private final double kd = -0.0002;  Value that was being tested as of February 3, 2022.
+    private double kd = 0.00003;
+
+    private double setPointHood;
+    private double setPointShooter;
```

Comment:
This "duplicate" variable still exists alongside the "setpointShooterPID" variable and if I'm understanding the purposes correctly, it still looks like they are being mistakenly mixed and matched with each other. I suggest deleting the declaration here and updating any other existing references to use "setpointShooerPID" instead.